src/sentry/static/sentry/app/views/groupMerged/mergedItem.jsx

```diff
@@ -130,13 +127,74 @@ const MergedItem = createReactClass({
                     hideLevel
                   />
                 </FlowLayout>
-              </SpreadLayout>
+              </EventDetails>
             )}
-          </div>
+          </MergedEventList>
         )}
-      </div>
+      </MergedGroup>
     );
   },
 });
 
+const MergedGroup = styled('div')`
+  ${p => p.busy && 'opacity: 0.2'};
+`;
+
+const ActionColumn = styled(Flex)`
+  padding: 0 10px;
+  align-items: center;
+
+  input {
+    margin: 0;
+  }
+`;
+
+const Controls = styled(({expanded, ...props}) => (
+  <Flex justify="space-between" {...props} />
+))`
+  border-top: 1px solid ${p => p.theme.borderLight};
+  background-color: #f3f1f6;
+  padding: ${space(0.5)} 0;
+  ${p => p.expanded && `border-bottom: 1px solid ${p.theme.borderLight}`};
+
+  ${MergedGroup}:first-child & {
```

Comment:
Could this be under MergedGroup (or am I understanding incorrectly)? I find this a bit difficult to read

Reply:
Yeah it can, but the reasoning here is that we are styling `Control` (under a specific parent), and want to maintain where its style definitions are defined (within the component/styled component).